ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -0,0 +1,376 @@
+                visible: collectiblesView.count == 0
+                horizontalAlignment: Text.AlignHCenter
+                verticalAlignment: Text.AlignVCenter
+                color: Theme.palette.directColor1
+                text: qsTr("%1 doesn't have any collectibles/NFTs yet").arg(d.mainDisplayName)
+            }
+
+            StatusGridView {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                id: collectiblesView
+                cellWidth: width/4
+                cellHeight: cellWidth
+                visible: count
+                model: root.isCurrentUser ? root.walletStore.flatCollectibles : null // TODO show other users too
+                ScrollBar.vertical: StatusScrollBar { }
+                delegate: StatusRoundedImage {
+                    width: GridView.view.cellWidth - Style.current.smallPadding
+                    height: GridView.view.cellHeight - Style.current.smallPadding
+                    border.width: 1
+                    border.color: Theme.palette.directColor7
+                    color: !!model.backgroundColor ? model.backgroundColor : "transparent"
+                    radius: Style.current.radius
+                    showLoadingIndicator: !model.collectionCollectiblesLoaded
+                    image.fillMode: Image.PreserveAspectCrop
+                    image.source: model.imageUrl ?? ""
+
+                    RowLayout {
+                        anchors.left: parent.left
+                        anchors.leftMargin: Style.current.halfPadding
+                        anchors.bottom: parent.bottom
+                        anchors.bottomMargin: Style.current.halfPadding
+                        anchors.right: parent.right
+                        anchors.rightMargin: Style.current.halfPadding
+
+                        Control {
+                            Layout.maximumWidth: parent.width
+                            horizontalPadding: Style.current.halfPadding
+                            verticalPadding: Style.current.halfPadding/2
+                            visible: !!model.id
+
+                            background: Rectangle {
+                                radius: Style.current.halfPadding/2
+                                color: Theme.palette.indirectColor2
+                            }
+                            contentItem: StatusBaseText {
+                                font.pixelSize: 13
+                                font.weight: Font.Medium
+                                maximumLineCount: 1
+                                elide: Text.ElideRight
+                                text: `#${model.id}`
+                            }
+                        }
+                    }
+
+                    StatusToolTip {
+                        visible: hhandler.hovered && (!!model.name || !!model.description)
+                        text: `<b>${model.name}</b><br/>${model.description}`
```

Comment:
The tooltip doesn't resize properly if the name or description is missing.
https://user-images.githubusercontent.com/47811206/223197464-8c091382-bf77-4e9e-b1a0-0b61ab89fd66.mov

Reply:
Good catch

Reply:
Fixed